Config/common/ai_strategy.cwt

```diff
@@ -171,6 +171,129 @@ alias[ai_strategy_rule:ai_strategy] = {
 	id = <building>
 	value = int
 }
+alias[ai_strategy_rule:ai_strategy] = {
+	type = put_unit_buffers
+	
+	ratio = float[0..1]
+	
+	## cardinality = 0..1
+	order_id = int
+	
+	## cardinality = 0..1
+	states = {
+		## cardinality = ~1..inf
+		int
+	}
+	
+	## cardinality = 0..inf
+	area = <ai_area>
+	
+	## cardinality = 0..inf
+	subtract_invasions_from_need = bool
+	
+	## cardinality = 0..inf
+	subtract_fronts_from_need = bool
+}
+alias[ai_strategy_rule:ai_strategy] = {
+	type = front_control
+	
+	## cardinality = 0..inf
+	tag = enum[country_tags]
+	
+	## cardinality = 0..inf
+	state = int
+	
+	## cardinality = 0..inf
+	strategic_region = <strategic_region>
+	
+	## cardinality = 0..1
+	country_trigger = {
+		## cardinality = ~1..inf
+		alias_name[trigger] = alias_match_left[trigger]
+	}
+	
+	## cardinality = 0..1
+	## replace_scope = { this = state from = country fromfrom = country }
+	state_trigger = {
+		## cardinality = ~1..inf
```

Comment:
Cardinality currently does not affect aliases, but leave this here in case I ever find a way to add it